docs/mobile-apps/automated-testing/appium.md

```diff
@@ -1,437 +1,141 @@
-
-<br/>
-
-### 6. Run Tests and Record Your Results
-
-This is where you log whether your test passed, failed, or returned an error.
-
-Running tests and recording test results is the ultimate purpose of your test script: you run tests in an automated test script in order to evaluate function and performance in the AUT, without requiring human interaction.
-
-#### **Test Frameworks**
-
-To run test and to record test results, you use methods of a test framework for your programming language. There are many available test frameworks, including the frameworks in the so-called XUnit family, which includes:
-
-* JUnit for Java
-* NUnit for C#
-* unittest or pyunit for Python
-* RSpec for Ruby
-
-For some programming languages, test frameworks other than those in the XUnit family are common - for example, the RSpec framework for Ruby. Our [sample test framework repos on GitHub](https://github.com/saucelabs-training) contain over 60 examples of test frameworks set up to work with Sauce Labs. 
-
-#### **Test Assertions**
-
-Most test frameworks implement the basic concept of an ***assertion***, a method representing whether or not a logical condition holds after interaction with an AUT.
-
-Test frameworks generally declare methods whose names begin with the term `assert` and end with a term for a logical condition (e.g., `assertEquals` in JUnit). Generally, when the logical condition represented by an assert method does not hold, an exception for the condition is thrown.
-
-There are various approaches to using exceptions in most test frameworks. The [SeleniumHQ documentation](https://www.selenium.dev/documentation/en/) has more detailed information on using assertions and verifications in your tests. 
-
-#### **Recording Test Results**
-
-Recording of test results can be done in various ways, supported by the test framework or by a logging framework for the programming language, or by both together. Selenium also supports taking screenshots of web browser windows as a helpful additional type of recording.
-
-Because of the wide variations in recording technique, this beginning section omits recording, instead emphasizing a simple approach to applying a test using an assert method. See our [sample test framework repos](https://github.com/saucelabs-training) for examples of setting up test results reporting.
-
-The following example runs a test by asserting that the login response message is equal to an expected success message:
-
-```java
-import org.junit.jupiter.api.Assertions.assertEquals;
-
-WebElement messageElement     = driver.findElement(By.id("loginResponse"));
-String message                = messageElement.getText();
-String successMsg             = "Welcome to foo. You logged in successfully.";
-assertEquals (message, successMsg);
+platformName: "iOS"
+deviceName: "iPhone 12"
```

Comment:
This is an example for a Real device (simulators are always post fixed with ` Simulator`. Can we be consistent with our examples?

Reply:
My mistake- this isn't an EMU test, it's an RDC, I will correct that typo, should say 'Sauce Labs Real Devices' instead of 'Sauce Labs Emulators' in the paragraph at the top of the section, will do a commit to update.

Is this a good way to write an RDC cap? "iPhone 12" ? or should I add the word "Device" at the end?